CMakeLists.txt

```diff
@@ -533,28 +503,23 @@ foreach(header ${HEADERS_TO_CHECK})
     string(TOUPPER ${header} tmp)
 
     string(REGEX REPLACE [./] "_" upper_header ${tmp})
-    check_include_file(${header} HAVE_${upper_header})
+    set(header_flag "HAVE_${upper_header}")
```

Comment:
In Cmake, the variables have upper case and function have lower case. Let's stick to one convention

Reply:
Ok done!